src/Compilers/VisualBasic/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.vb

```diff
@@ -210,6 +210,27 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 symbol = symbol.TupleUnderlyingType
             End If
 
+            ' It would be nice to handle C# NoPia symbols too, but it's not worth the effort.
+
+            Dim illegalGenericInstantiationSymbol = TryCast(symbol, NoPiaIllegalGenericInstantiationSymbol)
```

Comment:
I don't remember exactly which ones. The NoPia tests were printing the error types as "?" in the error messages rather than giving the name of the underlying type. This is code I ported, which was already in the C# code base and simply missing from the VB code base.